### PR TITLE
"An instance of X" phpdocs removal

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

That's #32973 on 4.4 :P

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Thank you @fabpot.

File: src/Symfony/Component/Security/Http/Firewall/LogoutListener.php [hunk 1]
```diff
@@ -42,11 +42,8 @@ class LogoutListener implements ListenerInterface
     private $csrfTokenManager;
 
     /**
-     * @param TokenStorageInterface          $tokenStorage
-     * @param HttpUtils                      $httpUtils        An HttpUtils instance
-     * @param LogoutSuccessHandlerInterface  $successHandler   A LogoutSuccessHandlerInterface instance
-     * @param array                          $options          An array of options to process a logout attempt
-     * @param CsrfTokenManagerInterface|null $csrfTokenManager A CsrfTokenManagerInterface instance
+     * @param TokenStorageInterface $tokenStorage
```
Comment (Tobion) [inline src/Symfony/Component/Security/Http/Firewall/LogoutListener.php]: useless leftover

File: src/Symfony/Component/HttpKernel/HttpKernel.php [hunk 2]
```diff
@@ -210,9 +208,8 @@ private function finishRequest(Request $request, int $type)
     /**
      * Handles an exception by trying to convert it to a Response.
      *
-     * @param \Exception $e       An \Exception instance
-     * @param Request    $request A Request instance
-     * @param int        $type    The type of the request (one of HttpKernelInterface::MASTER_REQUEST or HttpKernelInterface::SUB_REQUEST)
+     * @param \Exception $e    An \Exception instance
```
Comment (Tobion) [inline src/Symfony/Component/HttpKernel/HttpKernel.php]: useless leftover

File: src/Symfony/Component/Security/Http/Firewall/LogoutListener.php (hunk 1 above)
Comment (Tobion) [reply src/Symfony/Component/Security/Http/Firewall/LogoutListener.php]: #33018

File: src/Symfony/Component/HttpKernel/HttpKernel.php (hunk 2 above)
Comment (Tobion) [reply src/Symfony/Component/HttpKernel/HttpKernel.php]: #33018

**Pull request merged**